### PR TITLE
[Imagery Plugin] Data integration facilitation

### LINKED ISSUE #3394: [Images] Add helper class and metadata attributes for VERVE integration
VERVE requires some imagery metadata to be provided from Open MCT when an image is clicked on.

1. Add an additional `js-verve-image` class to [the div element that contains the image in the imagery view](https://github.com/nasa/openmct/blob/master/src/plugins/imagery/components/ImageryViewLayout.vue#L25) to allow VERVE to easily select the image element and attach a click event listener. 
2. Add new data attributes `data-openmct-image-timestamp` and `data-openmct-object-keystring`. These will allow VERVE to retrieve the timestamp of the image, and the ID of the image object (which contains the camera name).

This will allow VERVE code to add a click event listener to image elements, and retrieve metadata like so:

```
document.querySelectorAll('.j-verve-image').forEach((imageElement) => {
    imageElement.addEventListener('click' () => {
        const timestamp = imageElement.dataset.openmctImageTimestamp;
        const camera = imageElement.dataset.openmctObjectKeystring.split('~')[2].match(/(.*)_image_url/)[1];
    });
});
```

### PR DESCRIPTION
**Pull request by jvigliotta**
Added two data attributes to the main imagery div. One for the telemetry object's keystring and one for the main images timestamp. Also added a class to facilitate targeting the div to get access to these data attributes.

closes #3394

## Author Checklist
Changes address original issue? yes
Unit tests included and/or updated with changes? pending imagery test spec
Command line build passes? yes
Changes have been smoke-tested? yes
Testing instructions included? yes

### CODE REVIEW

File: src/plugins/imagery/components/ImageryViewLayout.vue [hunk 1]
```diff
@@ -22,10 +22,12 @@
                 ></a>
             </span>
         </div>
-        <div class="main-image s-image-main c-imagery__main-image has-local-controls"
+        <div class="main-image s-image-main c-imagery__main-image has-local-controls j-imagery-image"
```
Comment (akhenry) [inline src/plugins/imagery/components/ImageryViewLayout.vue:25]: Can we rename this js-imageryView-image? My bad I think

**Review by akhenry — CHANGES_REQUESTED** [review]
Looks great, just a really minor issue to address.

File: src/plugins/imagery/components/ImageryViewLayout.vue (hunk 1 above)
Comment (jvigliotta) [reply src/plugins/imagery/components/ImageryViewLayout.vue]: haha, I tried to clear it with you! Updated : )

Comment (akhenry) [reply src/plugins/imagery/components/ImageryViewLayout.vue]: @charlesh88 Pointed out that the correct prefix is actually `js-`, not `j-`. I messed up in my original naming suggestion.

**Review by akhenry — APPROVED** [review]

**Pull request merged**